spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -372,16 +375,16 @@ will emit a message with a TaskLaunchRequest object containing the required laun
 would be to launch the timestamp task once every 5 seconds, the stream to implement this would look like:
 
 ```
-stream create foo --definition "triggertask --triggertask.uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.0.0.BUILD-SNAPSHOT --trigger.fixed-delay=5 | task-launcher-local" --deploy
+stream create foo --definition "triggertask --triggertask.uri=<uriToJar> --trigger.fixed-delay=5 | task-launcher-local" --deploy
```

Comment:
Can we list a specific jar file here that we know will work.  ATM, this is the only "docs" on how to use the trigger task, so doing a cut-n-paste here so a user can get something working would be good.  Perhaps forward code the RELEASE coordinates of task app starters to be done right after the release of SCDF.